Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -1930,6 +1927,31 @@ private void OnDeviceChange(InputDevice device, InputDeviceChange change)
             }
         }
 
+        private bool IsCurrentControlScheme(ref InputControlScheme scheme, InputDevice[] devices)
+        {
+            var isSameScheme = user.controlScheme.HasValue && user.controlScheme.Value.Equals(scheme);
+            return (isSameScheme &&
+                ArrayHelpers.EqualSets(user.pairedDevices.ToArray(), devices) &&
+                !user.hasMissingRequiredDevices);
+        }
+
+        private void SwitchControlSchemeInternalUnchecked(ref InputControlScheme controlScheme, params InputDevice[] devices)
```

Comment:
Sorry for the confusion, is an old habit to emphasize internal functions that doesn't implement preconditional contracts since they are already indirectly given by calling code (to avoid doing it twice). I will rename this in a coming commit.